CMake/External_OpenCV.cmake

```diff
@@ -82,6 +82,21 @@ else()
 endif()
 
 
+if(fletch_ENABLE_OpenBLAS)
+  if (OpenCV_version VERSION_LESS 3.2.0)
```

Comment:
Why not just use 
`if (NOT OpenCV_version VERSION_LESS 3.2.0)`
and skip the empty if statement?

Reply:
Because my knowledge of CMake Scripting syntax is a bit lacking, but now it is less so.